lib/helpers/focusManager.js

```diff
@@ -1,47 +1,48 @@
-var findTabbable = require('../helpers/tabbable');
-var modalElement = null;
-var focusLaterElement = null;
-var needToFocus = false;
+import findTabbable from './tabbable';
 
-function handleBlur(event) {
+let modalElement = null;
+let focusLaterElement = null;
+let needToFocus = false;
+
+function handleBlur () {
   needToFocus = true;
 }
 
-function handleFocus(event) {
+function handleFocus () {
   if (needToFocus) {
     needToFocus = false;
     if (!modalElement) {
       return;
     }
     // need to see how jQuery shims document.on('focusin') so we don't need the
     // setTimeout, firefox doesn't support focusin, if it did, we could focus
-    // the element outside of a setTimeout. Side-effect of this implementation 
-    // is that the document.body gets focus, and then we focus our element right 
+    // the element outside of a setTimeout. Side-effect of this implementation
+    // is that the document.body gets focus, and then we focus our element right
     // after, seems fine.
-    setTimeout(function() {
-      if (modalElement.contains(document.activeElement))
+    setTimeout(() => {
+      if (modalElement.contains(document.activeElement)) {
         return;
-      var el = (findTabbable(modalElement)[0] || modalElement);
+      }
+      const el = (findTabbable(modalElement)[0] || modalElement);
       el.focus();
     }, 0);
   }
 }
 
-exports.markForFocusLater = function() {
+exports.markForFocusLater = function markForFocusLater () {
```

Comment:
oops, I think `export` keyword must be enabled. So, it can be setup or keep the `exports.markForFocusLater`.